addons/web_editor/static/src/js/editor/toolbar.js

```diff
@@ -18,6 +18,7 @@ export class Toolbar extends Component {
         showChecklist: { type: Boolean, optional: true },
         showColors: { type: Boolean, optional: true },
         showFontSize: { type: Boolean, optional: true },
+        customFontSize: { type: Boolean, optional: true },
```

Comment:
The name is not very clear. Maybe `useFontSizeInput`?

Reply:
@Zynton changes made. Cheers!